src/fat/image.rs

```diff
@@ -73,6 +74,19 @@ impl Image {
         Ok(())
     }
 
+    pub fn bios_parameter(&self) -> BIOSParam {
+        let params: BIOSParam;
+        let mut bios_bytes: [u8; 13] = [0; 13];
+        bios_bytes.clone_from_slice(&self.boot_sector[11..24]);
+        params = unsafe {
+            let mut param = ::std::mem::zeroed();
+            let p = &mut param as *mut BIOSParam as *mut [u8; 13];
+            ::std::ptr::write(p, bios_bytes);
+            param
+        };
```

Comment:
The #[repr(C)] 's around the structs guarantee the struct's wont be reordered.

We aren't interested in the first few bytes (boot code), so this let us skip to the "good stuff" in boot_sector without needing to hack it up.

Definitely open to better solutions, after an hour or so talking to #rust in irc, this was their best solution :-)  I noticed the field seemed to be misaligned... but that behavior only occurred in one image, so I was hoping something in the image was wrong.